mii/client.py

```diff
@@ -8,21 +8,38 @@
 import mii
 from mii.utils import get_task
 from mii.grpc_related.proto import modelresponse_pb2, modelresponse_pb2_grpc
-from mii.constants import GRPC_MAX_MSG_SIZE, Tasks
+from mii.constants import GRPC_MAX_MSG_SIZE, Tasks, DeploymentType
 from mii.method_table import GRPC_METHOD_TABLE
-
-
-def _get_deployment_info(deployment_name):
-    configs = mii.utils.import_score_file(deployment_name).configs
-    task = configs[mii.constants.TASK_NAME_KEY]
-    mii_configs_dict = configs[mii.constants.MII_CONFIGS_KEY]
-    mii_configs = mii.config.MIIConfig(**mii_configs_dict)
-
-    assert task is not None, "The task name should be set before calling init"
-    return task, mii_configs
-
-
-def mii_query_handle(deployment_name):
+from mii.deployment import allocate_processes, create_score_file
+from mii.config import DeploymentConfig
+
+
+def _get_deployment_configs(deployment_tag):
+    deployments = {}
+    configs = mii.utils.import_score_file(deployment_tag).configs
+    for deployment in configs[mii.constants.DEPLOYMENTS_KEY].values():
+        deployment[mii.constants.DEPLOYED_KEY] = True
```

Comment:
What is the need for this hardcoded `True`? This seems like something we should be able to remove